test/forcing/NetCDFPerFeatureDataProvider_Test.cpp

```diff
@@ -51,7 +51,18 @@ void NetCDFPerFeatureDataProviderTest::TearDown()
 //Construct a forcing object
 void NetCDFPerFeatureDataProviderTest::setupForcing()
 {
-    nc_provider = std::make_shared<data_access::NetCDFPerFeatureDataProvider>("/local/ngen/data/huc01/huc_01/forcing/netcdf/huc01.nc", utils::getStdErr() );
+    char tmp[256];
+    getcwd(tmp, 256);
+    cout << "Current working directory: " << tmp << endl;
```

Comment:
In the test?

Reply:
Subjective :-j This arose from the code that is in there to try multiple cwd's. Purpose was troubleshooting the test running in an unexpected place... because the test fails or succeeds depending on where you run it... and different environments result in different default cwd's.